submissions/david/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="css/style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>IA Izumo ChatBot</title>
```

Comment:
Little typo here, we call AI the same way in French ^__^


```suggestion
    <title>AI Izumo ChatBot</title>
```